### PR TITLE
Add experimental information about the specs

### LINKED ISSUE #138: Consider splitting out experimental packages
A few areas (metrics, semantic conventions, opentracing shim) are still experimental and should be offered as separate packages (with either a `0.x` version or a `-experimental`/`-alpha` suffix).

Related: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md#signal-lifecycle

### PR DESCRIPTION
**Pull request by nachoBonafonte**
As talked yesterday at the SIG meeting, using separate versions number or packages for areas of the projects is extremely painful on Swift. We have added extra information to the readme and renamed some libraries onluy dependant on those experimental API. I hope it is enough to solve the issue.
Fixes #138

### CODE REVIEW

**Comment by nachoBonafonte** [comment]
Also as decided at the SIG meeting yesterday move Opentracing shim, and SwiftMetrics shim to a new importers library

File: Package.swift [hunk 1]
```diff
@@ -53,12 +53,15 @@ let package = Package(
         ),
         .target(name: "OpenTracingShim",
                 dependencies: ["OpenTelemetrySdk",
-                               "Opentracing"]
+                               "Opentracing"],
+                path: "Sources/Importers/OpenTracingShim"
         ),
         .target(name: "SwiftMetricsShim",
```
Comment (Sherlouk) [inline Package.swift:59]: Should we also rename these to importers? `SwiftMetricsImporter` and `OpenTracingImporter`

Comment (nachoBonafonte) [reply Package.swift:59]: "Opentracing shim" name is used everywhere in the opentelemetry docs

Comment (Sherlouk) [reply Package.swift:59]: Fair enough, happy to leave - no worries 👍

**Comment by nachoBonafonte** [comment]
friendly ping @SergeyKanzhelev

**Review by SergeyKanzhelev — APPROVED** [review]

**Pull request merged**